guide/authentication/index.md

```diff
@@ -94,18 +113,39 @@ starting point.
 
 Basic Authentication
 --------------------
-Basic Authentication is an optional authentication handler for external clients.
-Due to the complexity of OAuth authentication, Basic authentication can be
-useful during development. However, Basic authentication requires passing your
+Basic authentication is an optional authentication handler for external clients.
+Due to the complexity of OAuth authentication, basic authentication can be
+useful during development. However, basic authentication requires passing your
```

Comment:
If basic is capitalized then it is a proper noun, which would means authentication is also capitalized as well, since that's the whole term.